### DISCUSSION TITLE
Option for inverted target for wire:loading

### DISCUSSION THREAD

**Discussion by patriktoth67** [livewire/livewire#6661] [3. Feature ideas] [answered]
Hi guys!

I'm finding myself more and more in need of an inverted wire:target, which would basically do the opposite of wire:target:
- the wire:loading fires for everything, except the functions/variables listed.

Why would I need this?

I have a page, where users can filter based on tons of variables. Each time something gets updated, I display a loading indicator. But there is one button, which runs a function which then redirects to another page, and displaying the same loading indicator for that doesn't make sense.

It can be solved of course by adding all the variables to a wire:target, but it'd be great to have to type only the one inverted function.

**Comment by whousehomestay** [comment]
Is there any PR to this?

**Reply by patriktoth67** [reply]
I just made one, not accepted yet: #8065

**Reply by patriktoth67** [reply, marked as answer]
v3.4.9 includes it now

**Reply by whousehomestay** [reply]
Very nice!!! Thank you ^_^
